actors/paych/tests/paych_actor_test.rs

```diff
@@ -214,6 +214,7 @@ mod create_lane_tests {
         let payer_addr = Address::new_id(PAYER_ADDR);
```

Comment:
nit: remove the unused (both here and in go tests) _balance and _received fields in the TestCase struct